### ISSUE TITLE
remove our views on package unload

### ISSUE THREAD

**Issue by braver** [SublimeLinter/SublimeLinter#1148]
When you disable SublimeLinter, the highlights and the statusbar, etc. remain. We should clean up right before the package unloads.

**Comment by kaste** [comment]
The implementation here is to look into each file ending with `_view.py` and write a `plugin_unloaded` function which then cleans its own space. This can be done in baby-steps, one after the other.

**Comment by braver** [comment]
Yeah, so I'm taking all the good-first-issue features because I'm a total noob and that way I can feel like I actually contribute something useful 🌵
